tests/clar.c

```diff
@@ -124,6 +125,8 @@ static struct {
 	int exit_on_error;
 	int report_suite_names;
 
+	int verbose;
```

Comment:
As this doesn't only specify whether it's verbose or not but also how verbose it is, I think like `verbosity` would be a better name here